engine/polyglot-api-macros/src/main/scala/org/enso/polyglot/macros/SerdeConfig.scala

```diff
@@ -0,0 +1,41 @@
+package org.enso.polyglot.macros
+
+import com.github.plokhotnyuk.jsoniter_scala.macros._
+import com.github.plokhotnyuk.jsoniter_scala.core._
```

Comment:
It does, yes. We could disallow grouping of imports so that it is in the same style as Java but it will create a lot of changes and disturb the purpose of this PR.